### PR TITLE
Feat/domain driven

### PR DESCRIPTION
**Pull request by yungxramoz**
- Bootify Updated
- Added all Entities
- Restructure to Domain Driven
- Auth Domain added

### CODE REVIEW

File: apps/backend/README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ To view the Bootify.io Setup, visit [this link](https://bootify.io/app/1AFNHOA9N
 
 ## Development
 
-During development it is recommended to use the profile `local`. In IntelliJ `-Dspring.profiles.active=local` can be
-added in the VM options of the Run Configuration after enabling this property in "Modify options". Create your own
+During development it is recommended to use the profile `local`. In IntelliJ `--spring.config.additional-location=classpath:application.local.yml --spring.profiles.active=local` can be
```
Comment (shoedler) [inline apps/backend/README.md:9]: Ha jetz gd dr ander PR gmerged - iz isches `application-development.yml` - und z "profile" heisst nöi o nüm `local` sondern `development`

File: apps/backend/src/main/java/tabletop/gather/backend/game/GameService.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import org.springframework.stereotype.Service;
+import tabletop.gather.backend.user.UserRepository;
+import tabletop.gather.backend.util.NotFoundException;
+
+import java.util.List;
+import java.util.UUID;
+
+
+@Service
+@Transactional
```
Comment (shoedler) [inline apps/backend/src/main/java/tabletop/gather/backend/game/GameService.java:14]: Wieso `@Transactional`? Isch ds vo bootify?

File: apps/backend/src/main/java/tabletop/gather/backend/user/User.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    @Id
+    @Column(nullable = false, updatable = false)
+    @GenericGenerator(name = "uuid", strategy = "org.hibernate.id.UUIDGenerator")
+    @GeneratedValue(generator = "uuid")
+    private UUID id;
+
+    @Column(nullable = false, length = 50)
+    private String username;
+
+    @Column(nullable = false)
```
Comment (shoedler) [inline apps/backend/src/main/java/tabletop/gather/backend/user/User.java:40]: Die wäre beidi no `length = 255`

File: apps/backend/src/main/java/tabletop/gather/backend/config/RestExceptionHandler.java [hunk 4]
```diff
@@ -28,16 +26,6 @@ public ResponseEntity<ErrorResponse> handleNotFound(final NotFoundException exce
         return new ResponseEntity<>(errorResponse, HttpStatus.NOT_FOUND);
     }
 
-    @ExceptionHandler(DataIntegrityViolationException.class)
```
Comment (shoedler) [inline apps/backend/src/main/java/tabletop/gather/backend/config/RestExceptionHandler.java:31]: Wieso chunnt di wäg? I bruche die bim Login im Frontend

**Review by shoedler — CHANGES_REQUESTED** [review]

File: apps/backend/src/main/java/tabletop/gather/backend/game/GameService.java (hunk 2 above)
Comment (yungxramoz) [reply apps/backend/src/main/java/tabletop/gather/backend/game/GameService.java:14]: Ja chunnt vo Bootify

File: apps/backend/src/main/java/tabletop/gather/backend/user/User.java (hunk 3 above)
Comment (yungxramoz) [reply apps/backend/src/main/java/tabletop/gather/backend/user/User.java:40]: length = 255 isch default, i nihme das no bim passwordHash use

File: apps/backend/src/main/java/tabletop/gather/backend/user/User.java [hunk 5]
```diff
@@ -43,7 +43,7 @@ public class User implements UserDetails {
     @Column(nullable = false)
     private String lastName;
```
Comment (shoedler) [inline apps/backend/src/main/java/tabletop/gather/backend/user/User.java:44]: `lastName` wär o 255

File: apps/backend/README.md [hunk 6]
```diff
@@ -6,6 +6,8 @@ To view the Bootify.io Setup, visit [this link](https://bootify.io/app/1AFNHOA9N
 
 ## Development
 
+### Prerequisites
+
 During development it is recommended to use the profile `development`. In IntelliJ `--spring.config.additional-location=classpath:application-development.yml --spring.profiles.active=development` can be
 added in the Run Configuration after enabling this property in "Modify options". Create your own
 `application-development.yml` file to override settings for development.
```
Comment (shoedler) [inline apps/backend/README.md:13]: I ha no dr folder Pfad ergänzt gha: Create your own `application-development.yml` in file in the `apps\backend\src\main\resources\` directory to override settings for development

**Review by shoedler — APPROVED** [review]

**Pull request merged**